tools/accuracy_checker/accuracy_checker/preprocessor/geometric_transformations.py

```diff
@@ -435,7 +440,7 @@ def process(self, image, annotation_meta=None):
         pref_width = math.ceil(pref_width / float(self.stride)) * self.stride
         pad = self.pad_func(pref_width, pref_height, width, height)
         image.metadata['padding'] = pad
-        padding_realization_func = self._opencv_padding if not self.use_numpy else self._numpy_padding
+        padding_realization_func = self._opencv_padding if not self.use_numpy else partial(self._numpy_padding, mode=self.numpy_pad_mode)
```

Comment:
`self._numpy_padding` is method and has access to class instance fields, So you are able directly use `self.numpy_pad_mode` inside it without providing it as argument and partial function creation.

Reply:
> `self._numpy_padding` is method and has access to class instance fields, So you are able directly use `self.numpy_pad_mode` inside it without providing it as argument and partial function creation.

Hi,
The numpy_padding is called by:
image.data = padding_realization_func(image.data, pad)
here the mode is not passed in, because it also supports opencv padding, which does not have the mode parameter.

Reply:
no, I mean what prevents to do something like this?
```python
    def _numpy_padding(self, image, pad):
        pad_values = (
            (self.pad_value[0], self.pad_value[0]),
            (self.pad_value[1], self.pad_value[1]),
            (self.pad_value[2], self.pad_value[2])
        )
        if self.numpy_pad_mode != 'constant':
            return np.pad(
                image, ((pad[0], pad[2]), (pad[1], pad[3]), (0, 0)),
                mode=elf.numpy_pad_mode
            )
        else:
            return np.pad(
                image, ((pad[0], pad[2]), (pad[1], pad[3]), (0, 0)),
                mode=elf.numpy_pad_mode, constant_values=pad_values
            )
```

Reply:
Understand. I'll modify it.

Reply:
OK, I understand. This will work using self.numpy_pad_mode